packages/shared/src/components/squads/Comment.tsx

```diff
@@ -54,13 +60,20 @@ export function SquadComment({
   onUpdateForm,
 }: SquadCommentProps): ReactElement {
   const textinput = useRef<HTMLTextAreaElement>();
-  const { url, post: postItem } = form;
+  const { privateLink, post: postItem } = form;
   const { post } = postItem;
   const { user } = useContext(AuthContext);
   const [commentary, setCommentary] = useState(form.commentary);
-  const isLink = !isNullOrUndefined(url);
-  const [link, setLink] = useState(url);
-  const [linkHint, setLinkHint] = useState(url);
+  const [link, setLink] = useState(privateLink?.url);
+  const [linkHint, setLinkHint] = useState(privateLink?.url);
+  const { mutateAsync: getPrivateLink } = useMutation(getExternalLinkPreview, {
+    onSuccess: (preview, url) => {
+      onUpdateForm({ privateLink: { url, ...preview } });
+    },
+    onError: (_, url) => {
+      onUpdateForm({ privateLink: { url } });
```

Comment:
Should we show a toast notification for when the limit has been reached?

Reply:
I think so. Make sure to validate that it's indeed this error, if it's another one ask the user to try again